### PR TITLE
feature: compare after measure

### LINKED ISSUE #57: [FEATURE] display compare results to console after `reassure measure` if both baseline & current results are present
**Is your feature request related to a problem? Please describe.**
When working locally user has to issue separate steps of `reassure measure` and `reassure compare` to get the measurement results.

**Describe the solution you'd like**
If after running `reassure measure` there are both current and baseline performance measurements, then automatically display compare results to the console. Probably not worth generating MD & JSON output in such cases.

Add `--no-compare` flag to disable the behaviour (i.e. just measure and do not display compare results to console).

**Describe alternatives you've considered**
1. Not implement it at all and require separate `measure` and `compare` for local workflow

**Additional context**
V8 test suite behaves in a similar manner, they do not even have separate `compare` script in their normal workflow but just do it with the current code `measure` setup.

### PR DESCRIPTION
**Pull request by mdjastrzebski**
### Summary

Merges (removes) `reassure compare` with `reassure measure`. This also makes `reassure measure` useful as the default CLI command, so it can be run using just `reassure`.

Also: improves console output.

Resolves #57 

### Test plan

Tests run successfully.

### CODE REVIEW

**Comment by mdjastrzebski** [comment]
I wonder if we actually should remove the `compare` command altogether. User will basically have to run only these two commands in his perf script:
1. `reassure --baseline`
2. `reassure` (will also output compare, as baseline file is available.

Instead of `reassure measure  --baseline` + `reassure measure` + `reassure compare`.

@thymikee @Xiltyn wdyt?

**Comment by mdjastrzebski** [comment]
For reference V8 does this:

Baseline
```
$ test/benchmarks/csuite/csuite.py sunspider baseline out.gn/master/d8
Wrote ./_results/master.
Run sunspider again with compare mode to see results.
```

Current + compare
```
$ test/benchmarks/csuite/csuite.py sunspider compare out.gn/x64.release/d8

                               benchmark:    score |   master |      % |
===================================================+==========+========+
                       3d-cube-sunspider:     13.9 S     13.4 S   -3.6 |
                      3d-morph-sunspider:      8.6 S      8.4 S   -2.3 |
                   3d-raytrace-sunspider:     15.1 S     14.9 S   -1.3 |
...
```

See: https://v8.dev/docs/benchmarks#example-usage

**Comment by mdjastrzebski** [comment]
Check for #78 for implementation of the above concept

File: packages/reassure-cli/src/commands/measure.ts [hunk 1]
```diff
@@ -1,18 +1,24 @@
...
 const RESULTS_FILE = '.reassure/current.perf';
 const BASELINE_FILE = '.reassure/baseline.perf';
 
 type MeasureOptions = {
   baseline: boolean;
+  compare: boolean;
 };
 
 export function run(options: MeasureOptions) {
+  console.log('\n❇️  Running performance tests:');
```
Comment (thymikee) [inline packages/reassure-cli/src/commands/measure.ts:17]: TODO: make it a logger function so we can opt-out from logging some messages (make it an issue please)

File: packages/reassure-cli/src/commands/measure.ts [hunk 2]
```diff
@@ -28,17 +34,47 @@ export function run(options: MeasureOptions) {
...
+  if (existsSync(outputFile)) {
+    console.log(`\n✅  Written ${measurementType} performance measurements to ${outputFile}`);
+    console.log(`🔗 ${resolve(outputFile)}\n`);
+  } else {
+    console.error(`❌  Something went wrong, ${measurementType} performance file (${outputFile}) does not exist`);
+    return;
+  }
+
+  if (options.baseline) {
+    console.log("You can now run 'reassure' to measure & compare performance against modified code.");
```
Comment (thymikee) [inline packages/reassure-cli/src/commands/measure.ts:47]: make it a hint maybe?

**Comment by thymikee** [comment]
@mdjastrzebski I've merged the followup PR here. Can you adjust the description and changeset message?

File: packages/reassure-cli/src/commands/measure.ts (hunk 2 above)
Comment (mdjastrzebski) [reply packages/reassure-cli/src/commands/measure.ts]: What do you mean by that?

Comment (thymikee) [reply packages/reassure-cli/src/commands/measure.ts]: something like this
```diff
You can now run 'reassure' to measure & compare performance against modified code.
Hint: You can now run 'reassure' to measure & compare performance against modified code.
```

**Review by thymikee — APPROVED** [review]

**Pull request merged**